css/subject.styl

```diff
@@ -2,9 +2,10 @@
   min-width: 0
 
 .subject
+  margin: 0 auto
+  max-width: 800px
   .subject-container
```

Comment:
Looks like that css class name was used elsewhere, but didn't conflict. I just renamed this instance to `.subject-page` so it doesn't come up in the future